src/vma/vma_extra.h

```diff
@@ -421,6 +543,37 @@ struct __attribute__ ((packed)) vma_api_t {
 	 * @return 0 on success, or error code on failure.
 	 */
 	int (*dump_fd_stats) (int fd, int log_level);
+
+	/**
+	 * Get data from the MP_RQ cyclic buffer
+	 * @param fd - the fd of the ring to query - get it using @ref get_socket_rings_fds
+	 * @param completion results see @ref struct vma_completion_mp_t
+	 * @param min min number of packet to return, if not available
+	 * 	will return 0 packets
+	 * @param max max packets to return
+	 * @param flags can be MSG_DONTWAIT, MSG_WAITALL, MSG_PEEK
```

Comment:
Please change: `MSG_WAITALL, MSG_PEEK` --> `MSG_WAITALL (not yet supported), MSG_PEEK (not yet supported)`

#### Introduced in:
`add cyclic function api and enalbe related functions when not using VMA_POLL`